tests/zulemaArteaga/test_GX3_110_ToolsQA_PracticeForm.py

```diff
@@ -0,0 +1,68 @@
+from tests.testbase import *
+
+class Test_GX3_118_ToolsQA_TextBox:
+    
+    @pytest.fixture
+    def precondition(self, setWebDriver: WebDriver):
+        global web, get, do
+        web = setWebDriver
+        get = Locators(web)
+        do = Actions_to_execute(web)
+
+        get.page("https://demoqa.com/automation-practice-form")
+        
+        title = web.title
+        assert title == "DEMOQA"
+
+        yield
+        web.quit()
+
+    def test__fill_full_out_form(self, precondition):
+            '''TC01: Verify that after submiting the form, a popup window is displayed confirming the submited information'''
+            
+            with open('tests/zulemaArteaga/data/GX3_110_PracticeForm_data.csv', 'r') as file:
+                data = csv.DictReader(file)
+            
+            # Filling out the form with the data in file  
+                for row in data:
+                    name = do.fill_input_byID_identifier("firstName", row['name'])
+                    last_name = do.fill_input_byID_identifier("lastName", row['last_name'])
+                    email = do.fill_input_byID_identifier('userEmail', row['email'])
+                    
+                    # Selecting Radio Buttons to select gender provided in data file
+                    male = row['gender'] == "Male"
```

Comment:
male => is_male (is the correct way to name this kind of variables).

< is + adjective >
i.g isActive, isOpen, isDeprecated, isDeleted...